### PR TITLE
attach istio logging options to nodeagent cmd and add debug logs

### PR DESCRIPTION
**Pull request by quanjielin**
https://github.com/istio/istio/issues/5903

### CODE REVIEW

File: security/pkg/nodeagent/cache/secretcache.go [hunk 1]
```diff
@@ -150,6 +150,8 @@ func (sc *SecretCache) keyCertRotationJob() {
 }
 
 func (sc *SecretCache) rotate(t time.Time) {
+	log.Debug("Refresh job runs")
```
Comment (wattli) [inline security/pkg/nodeagent/cache/secretcache.go]: s/runs/running

File: security/pkg/nodeagent/sds/sdsservice.go [hunk 2]
```diff
@@ -120,12 +120,19 @@ func (s *sdsservice) StreamSecrets(stream sds.SecretDiscoveryService_StreamSecre
 				return receiveError
 			}
 
+			if discReq.Node == nil {
+				log.Errorf("Invalid discovery request, node is nil")
+				return fmt.Errorf("invalid discovery request nil node")
```
Comment (wattli) [inline security/pkg/nodeagent/sds/sdsservice.go]: invalid discovery request with no node

File: security/pkg/nodeagent/cache/secretcache.go (hunk 1 above)
Comment (quanjielin) [reply security/pkg/nodeagent/cache/secretcache.go]: done.

File: security/pkg/nodeagent/sds/sdsservice.go (hunk 2 above)
Comment (quanjielin) [reply security/pkg/nodeagent/sds/sdsservice.go]: done

**Pull request merged**